service/pom.xml

```diff
@@ -3,34 +3,20 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>tds-student-service</artifactId>
+    <groupId>org.opentestsystem.student</groupId>
+    <artifactId>tds-student-service-service</artifactId>
```

Comment:
I think we should rename the parent artifact to `tds-student`, that'll make it consistent with the other services (e.g. `tds-config` for TDS_ConfigService and `tds-exam` for TDS_ExamService.  Then this artifact should be named `tds-student-service`.